test/atm/global-workflow/config.base.emc.dyn

```diff
@@ -0,0 +1,402 @@
+#! /usr/bin/env bash
+
+########## config.base ##########
+# Common to all steps
+
+echo "BEGIN: config.base"
+
+# Machine environment
+export machine="@MACHINE@"
+
+# EMC parallel or NCO production
+export RUN_ENVIR="emc"
+
+# Account, queue, etc.
+export ACCOUNT="@ACCOUNT@"
+export QUEUE="@QUEUE@"
+export QUEUE_SERVICE="@QUEUE_SERVICE@"
+export PARTITION_BATCH="@PARTITION_BATCH@"
+export PARTITION_SERVICE="@PARTITION_SERVICE@"
+
+# Project to use in mass store:
+HPSS_PROJECT=emc-global
```

Comment:
probably not worth changing to emc-da?

Reply:
Since this is a copy of g-w `config.base.emc.dyn`, I suggest we keep it as `emc-global` for the time being.  Of course, I'm entirely fine with changing it to `emc-da`

Reply:
fine with me, will resolve